src/entry-server.tsx

```diff
@@ -22,6 +22,13 @@ export const render = async (options: RenderOptions): Promise<string> => {
   const { query } = createStaticHandler(routes, { basename: BASE_URL });
   const context = await query(options.request);
 
+  if (IS_DEBUG) {
+    const isHomePage = new URL(options.request.url).pathname.replace(BASE_URL, '') === '';
+    if (isHomePage) {
+      await options.i18n.changeLanguage(LanguageEnum.FR);
```

Comment:
why do we need this hack?

Reply:
I have to add a condition, because it is only in the case where we have the debug language

Reply:
And there is a rendering issue with 'debug language's only on the home page?